BoundingBox[] array
geometry_msgs/PoseStamped poseStamped